src/gc/env/gcenv.os.h

```diff
@@ -267,13 +336,13 @@ class GCToOSInterface
     // Check if the OS supports getting current processor number
     static bool CanGetCurrentProcessorNumber();
 
-    // Set ideal processor for the current thread
+    // Add ideal processor for the current thread
     // Parameters:
-    //  processorIndex - index of the processor in the group
-    //  affinity - ideal processor affinity for the thread
+    //  srcProcNo - processor number the thread currently runs on
+    //  dstProcNo - processor number the thread should be migrated to
     // Return:
     //  true if it has succeeded, false if it has failed
-    static bool SetCurrentThreadIdealAffinity(GCThreadAffinity* affinity);
+    static bool MigrateThread(uint16_t srcProcNo, uint16_t dstProcNo);
```

Comment:
the original name seem better 'cause you are setting an ideal processor, it's not guaranteed that the thread will be migrated to the other processor.

Reply:
The motivation to change the name was that I've added the srcProcNo parameter - I've moved code that was checking that we don't migrate between groups on Windows into this function.  Since we have two parameters now, the original name started to sound little strange. But you are right that I've overlooked the fact that on Windows, setting the ideal processor doesn't mean the thread is going to be migrated. I was influenced by the linux world where the affinity setting always migrates the thread. So while on Windows it is just a hint, on Linux it is a requirement.
I'll revert the name.